orttraining/orttraining/test/training_ops/cpu/tensor/gather_nd_grad_op_test.cc

```diff
@@ -74,7 +86,30 @@ TEST(GatherNDGradOpTest, GatherNDGrad_batch_dims_of_2) {
       });
   test.Run();
 }
-#endif
+
+TEST(GatherNDGradOpTest, GatherNDGrad_batch_dims_two_negative_indices) {
+  OpTester test("GatherNDGrad", 1, kMSDomain);
+  test.AddAttribute<int64_t>("batch_dims", 2);
+  test.AddInput<int64_t>("shape", {4}, {2, 2, 2, 3});
+  test.AddInput<int64_t>(
+      "indices", {2, 2, 1},
+      {
+          -1,  // batch 0
+          -1,  // batch 1
+          -2,  // batch 2
+          1,   // batch 3
```

Comment:
nit - would it look better to shift right by 1 space?

Reply:
I think this is already aligned, considering there is negative symbol in line 99.  Correct me if I misundertood you. Thank you! :)